src/ErrorReporting/prepend.inc

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
I just pushed a change to this branch in the docs parser which will skip files with no class/trait/interface rather than erroring. We should be able to change this file to `prepend.php` now.